src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx

```diff
@@ -53,6 +54,20 @@ class BackButton extends React.Component {
         ? t('Project')
         : t('Organization');
 
+    // if the user needs to setup 2fa as part of the org invite flow,
+    // send them back to accept the invite
+    let pendingInvite = Cookies.get('pending-invite');
+    let shouldGoBackToInvite = pendingInvite && !lastAppContext;
```

Comment:
I feel like "pending invite" should disregard `lastAppContext`, I suppose this isn't necessarily the case when you are already a member of other organizations... thoughts?

Reply:
I was mildly worried about cookies not getting deleted so added that check... not opposed to getting rid of it though. Would lastAppContext always be falsey coming from the invite flow, whether or not you're a member of another organization?

Reply:
Yeah if they come directly from the invite flow, it'll always be null